btp/btp.go

```diff
@@ -307,6 +307,15 @@ func (b *BTP) addRelayMessage(bu *chain.BlockUpdate, rps []*chain.ReceiptProof)
 				rm.HeightOfDst = guessHeightOfDst
 			}
 		}
+
+		// create rm.BlockProof if lastblockupdate smaller than BMV height
+		if b.lastBlockUpdate.Height <= b.bmcLinkStatus.Verifier.Height {
+			var err error
+			if rm.BlockProof, err = b.newBlockProof(b.lastBlockUpdate.Height, b.lastBlockUpdate.Header); err != nil {
```

Comment:
@trmaphi it is not correct: BlockProof should not be built at the time AddRelayMessage.
BlockProof only be built when UpdateRelayMessages